apps/dash-simple-image-processor/assets/swansea/SWANSE__-392e.txt

```diff
@@ -0,0 +1,90 @@
+LICENSE AGREEMENT
+
+The attached font 'SWANSEA' is Copyright [c] Roger White - all 
+rights reserved. In using this font it will be taken that you have read and 
+agreed to the following conditions:-
+
+DISTRIBUTION.
+
+The font archives must contain all of the files specified:-
+
+Postscript Type 1 package - SWANSE__.ZIP contains:-
+
+SWANSE__.AFM
+SWANSE__.PFB
+SWANSE__.PFM
+SWANSE__.INF
+SWANSE__.TXT
+
+TrueType package - SWANSE__.ZIP Contains:-
+
+SWANSE__.TTF
+SWANSE__.TXT
+
+This font has been made available as Type 1 and Truetype for the PC.
+Alteration of the character set of the font, and conversion to any 
+other font format is expressly forbidden.
+
+PAYMENT FOR USE
+
+Provided this font is used for personal purposes then no fee is payable to 
+the author for it's use. You may use this font yourself and may pass on 
+copies to your friends provided that the copy contains all of the files 
+from the original package. Distribution can be by electronic means or on 
+floppy disk.
+
+If this font is used for commercial purposes then the user agrees to make a 
+one off payment to the author of �25.00.

```

Comment:
I think it's a problem to include files with such a license in this repo. What do you think @nicholas-esterer @VeraZab ?